policy/lib/tekton/trusted_test.rego

```diff
@@ -86,11 +86,6 @@ test_data_errors if {
 			{"ref": "bad-effective-on", "effective_on": "not-a-date"},
 			{"ref": "bad-effective-on", "effective_on": "2024-01-01T00:00:00Z", "expires_on": "not-a-date"},
 		],
-		# this is allowed
-		"duplicated-entries": [
```

Comment:
I think we should keep this to make sure we don't accidentally break this use case in the future.